Dockerfile

```
@@ -1,11 +1,11 @@
-FROM mcr.microsoft.com/dotnet/sdk:6.0-jammy
+FROM mcr.microsoft.com/dotnet/sdk:7.0-jammy
 
 RUN dotnet --version
 
 # Setting the path up to allow .NET tools
 ENV PATH "$PATH:/root/.dotnet/tools"
 
-RUN dotnet tool install --global docfx --version 2.70.1
+RUN dotnet tool install --global docfx --version 2.70.4
```

```suggestion
RUN dotnet tool install --global docfx --version 2.71.0
```

Please update to 2.71.0, or merge main into this branch. We're at 2.71.0 on our main branch.

I'd do it myself but wasn't given permission to edit this PR branch.